codegen/src/botocore.rs

```diff
@@ -0,0 +1,4 @@
+#[cfg(feature = "serde_macros")]
+include!("botocore.rs");
```

Comment:
I'm not clear on what this is doing - is it including itself?  Maybe it's some serde thing I've forgotten.